packages/node/src/integrations/context.ts

```diff
@@ -476,3 +478,8 @@ function getCloudResourceContext(): CloudResourceContext | undefined {
     return undefined;
   }
 }
+
+export const createMissingInstrumentationContext = (pkg: string): MissingInstrumentationContext => ({
```

Comment:
let's move this out of here, this file is about the `Context` integration, which is different than the context property on the event 😅 

most overloaded term in sentry :p